xcp_d/interfaces/surfplotting.py

```diff
@@ -88,8 +88,10 @@ class _PlotSVGDataInputSpec(BaseInterfaceInputSpec):
         desc="TSV file with filtered motion parameters.",
     )
     mask = File(exists=False, mandatory=False, desc="Bold mask")
+    tmask = File(exists=False, mandatory=False, desc="Bold mask")
     seg_data = File(exists=False, mandatory=False, desc="Segmentation file")
     TR = traits.Float(default_value=1, desc="Repetition time")
+    dummyvols = traits.Float(default_value=0, desc="Dummy volumes to drop")
```

Comment:
```suggestion
    dummyvols = traits.Int(default_value=0, desc="Dummy volumes to drop")
```

Reply:
Shouldn't this be an integer instead of a float though? It's the number of volumes to drop, rather than the seconds to drop